lib/taurus/qt/qtgui/table/taurusvaluestable.py

```diff
@@ -384,6 +384,105 @@ def getReadValue(self, index):
         return self._rtabledata[index.row(), index.column()]
 
 
+class TaurusValuesSlicedArrayModel(TaurusValuesIOTableModel):
+    '''
+    Table model that allows representing 1D data in 2D format with optionally
+    different number of items per row
+    '''
+
+    def __init__(self, slices=[], parent=None):
+        self._parent = parent
```

Comment:
I am worried about possible circular references. I would try to avoid keeping a hard ref to parent